lib/pbench/test/functional/server/conftest.py

```diff
@@ -56,17 +59,8 @@ def register_test_user(oidc_admin: OIDCAdmin):
 
 
 @pytest.fixture
-def login_user(
-    server_client: PbenchServerClient, oidc_admin: OIDCAdmin, register_test_user
-):
+def login_user(server_client: PbenchServerClient, register_test_user):
     """Log in the test user and return the authentication token"""
-    oidc_endpoints = server_client.endpoints["openid"]
-    response = oidc_admin.user_login(
-        client_id=oidc_endpoints["client"], username="tester", password="123456"
-    )
-    auth_token = response["access_token"]
-    assert auth_token
-    json = {"username": "tester", "auth_token": auth_token}
-    server_client.username = "tester"
-    server_client.auth_token = auth_token
-    yield JSONMap(json)
+    server_client.login(USERNAME, PASSWORD)
+    assert server_client.auth_token
+    yield
```

Comment:
The `yield` doesn't send any value, and there is no code which gets executed once it control returns to it, so I don't think it accomplishes anything.  I would omit it and just let control return from the fixture body normally.